### ISSUE TITLE
Deconvolution initialisation

### ISSUE THREAD

**Issue by hh2712** [BVLC/caffe#5611]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Issue summary
Hi,
I am trying to fine tuning FCN32s network. However, I've inspected that my accuracies(overall accuracies, mean accuracies and mean IUs fluctuate in certain range and the values does not happy to increase at all).

In the paper, it described that "Final layer deconvolutional filters are fixed to bilinear interpolation , while intermediate upsampling layers are intialised to bilinear upsampling, and then learned", but in the train.prototxt the learning rate was set to 0 and there is only one deconvolution layer. I am a little bit confused here... 

It would be greatly appreciated if someone provide me with any insight of this issue.

Hanyi 

@shelhamer 

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):
